src/extras/helpers/VertexNormalsHelper.js

```diff
@@ -61,7 +61,7 @@ THREE.VertexNormalsHelper.prototype.update = ( function () {
 
 		var keys = [ 'a', 'b', 'c' ];
 
-		this.object.updateMatrixWorld( true );
+		this.object.updateMatrixWorld();
```

Comment:
If `object.matrixWorld` is lazily updated whenever it is accessed, why do we need to call `updateMatrixWorld` -- ever?